package.json

```diff
@@ -12,6 +12,7 @@
     "deploy:prod": "cross-env BUILD_ENV=production gulp deploy",
     "firestore:init": "firebase firestore:delete -r --all-collections -y && firebase functions:config:set schedule.enabled=true && firebase deploy --except hosting && babel-node ./internals/import-default-data",
     "firestore:copy": "babel-node ./internals/firestore-copy-data",
+    "firestore:update": "babel-node ./internals/import-default-data",
```

Comment:
isn't that going to create double entries in the collections?
maybe we need limited firestore:delete call

Reply:
It overrides old entries